lib/components/narrative/line-itin/itin-summary.js

```diff
@@ -99,6 +99,9 @@ export class ItinerarySummary extends Component {
     const minTotalFare = minTNCFare * 100 + transitFare
     const maxTotalFare = maxTNCFare * 100 + transitFare
 
+    const formattedStartTime = itinerary.startTime + timeOptions.offset
+    const formattedEndTime = itinerary.endTime + timeOptions.offset
```

Comment:
Rename. These are just time stamps now.

Reply:
Renamed in https://github.com/opentripplanner/otp-react-redux/pull/480/commits/b4b757b5e0da96045794b7a642d7271e79218c10